ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -117,10 +127,12 @@ struct ListCellButtonStyle: ButtonStyle {
         }
         return configuration.label
             .contentShape(Rectangle())
-            .frame(minHeight: height)
             .listRowInsets(EdgeInsets())
+            .padding(.top, tokens.horizontalCellPadding / 2)
```

Comment:
replace these 4 .padding calls with the padding(EdgeInsets(top,bottom,leading,trailing))?